app/views/decidim/verifications/sms/authorizations/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="wrapper">
+  <div class="row collapse">
+    <div class="row collapse">
+      <div class="columns large-8 large-centered text-center page-title">
+        <h1><%= t(".title") %></h1>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="columns large-6 medium-centered">
+        <div class="card">
+          <div class="card__content">
+            <%= decidim_form_for(@form, url: authorization_path(redirect_url: redirect_url)) do |form| %>
+              <div class="field">
+                <%= form.text_field :mobile_phone_number, disabled: true %>
+              </div>
+
+              <div class="actions">
+                <%= form.submit t(".send"), class: "button expanded", "data-disable-with" => "#{t('.send')}...", disabled: @form.mobile_phone_number.blank? %>
+              </div>
+            <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
I like this idea of showing the form to inform the user which phone number we got together with the button. Much more clear than showing a notice that the SMS was sent straight away. I would just add a side note below the button, for instance, to tell this was imported by the admin. Something like `This is the phone number that was imported by the admin. Please, reach out to them if it's not right`.